### PR TITLE
[T-299] Running the reporter in a seperate worker

### PR DESCRIPTION
**Pull request by DeMoorJasper**
# ↪️ Pull Request

This PR allows the workerfarm to have dedicated worker queues for certain processes and utilises it for the reporterrunner.

### CODE REVIEW

File: packages/core/workers/src/WorkerFarm.js [hunk 1]
```diff
@@ -86,12 +89,12 @@ export default class WorkerFarm extends EventEmitter {
...
 
-    if (!this.options.workerPath) {
-      throw new Error('Please provide a worker path!');
+    if (!this.options.workerPaths) {
+      throw new Error('Please provide worker paths!');
     }
 
     // $FlowFixMe this must be dynamic
-    this.localWorker = require(this.options.workerPath);
+    this.localWorker = require(nullthrows(this.options.workerPaths.main));
```
Comment (DeMoorJasper) [inline packages/core/workers/src/WorkerFarm.js]: Do we still need localWorkers?

File: packages/core/core/src/worker-report.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+registerCoreWithSerializer();
+
+export async function runReport(
+  workerApi: WorkerApi,
+  {
+    config,
+    opts,
+    event,
+  }: {|
+    config: ParcelConfig,
```
Comment (DeMoorJasper) [inline packages/core/core/src/worker-report.js]: use a reference for this?
`let {ref, dispose} = await workerfarm.createSharedReference(sharedValue);`

File: packages/core/core/src/worker-report.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    opts: PluginOptions,
```
Comment (DeMoorJasper) [inline packages/core/core/src/worker-report.js]: use a reference for this?
`let {ref, dispose} = await workerfarm.createSharedReference(sharedValue);`

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>packages/benchmarks/kitchen-sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 7.43s | +906.00ms ⚠️ |
| Cached | 5.77s | +2.29s ⚠️ |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.ff215454.webp | 102.94kb | +0.00b | 251.00ms | +75.00ms ⚠️ |
| dist/modern/parcel.76ee4591.webp | 102.94kb | +0.00b | 250.00ms | +75.00ms ⚠️ |
| dist/legacy/index.js | 252.00b | +0.00b | 515.00ms | +65.00ms ⚠️ |
| dist/modern/index.js | 215.00b | +0.00b | 510.00ms | +64.00ms ⚠️ |
| dist/modern/index.css | 29.00b | +0.00b | 1.19s | +121.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.ff215454.webp | 102.94kb | +0.00b | 22.00ms | +8.00ms ⚠️ |
| dist/modern/parcel.76ee4591.webp | 102.94kb | +0.00b | 18.00ms | +6.00ms ⚠️ |
| dist/legacy/index.js | 252.00b | +0.00b | 28.00ms | +11.00ms ⚠️ |
| dist/modern/index.js | 215.00b | +0.00b | 20.00ms | +6.00ms ⚠️ |
| dist/legacy/index.css | 29.00b | +0.00b | 18.00ms | +6.00ms ⚠️ |
| dist/modern/index.css | 29.00b | +0.00b | 16.00ms | +5.00ms ⚠️ |

</p></details>

<details><summary>packages/benchmarks/react-hn ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 12.46s | +2.84s ⚠️ |
| Cached | 6.04s | +2.47s ⚠️ |

#### Cold Bundles

*No bundle changes detected.*

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/module/index.js | 34.29kb | +0.00b | 20.00ms | +12.00ms ⚠️ |
| dist/main/index.js | 34.27kb | +0.00b | 25.00ms | +13.00ms ⚠️ |

</p></details>

<details><summary>packages/benchmarks/ak-editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 3.70m | +51.78s ⚠️ |
| Cached | 8.22s | +2.44s ⚠️ |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 2.33mb | +6.00b ⚠️ | 1.98m | +20.41s ⚠️ |
| dist/pdfRenderer.85e56a51.js | 1.11mb | +0.00b | 56.10s | +25.13s ⚠️ |
| dist/smartMediaEditor.678dca17.js | 601.49kb | +0.00b | 1.56m | +26.31s ⚠️ |
| dist/popup.cc1a1fa6.js | 285.42kb | +0.00b | 2.02m | +20.47s ⚠️ |
| dist/EmojiPickerComponent.7202df5c.js | 105.88kb | +0.00b | 1.09m | +26.65s ⚠️ |
| dist/Toolbar.5c7a6fbb.js | 105.61kb | +0.00b | 1.30m | +11.00s ⚠️ |
| dist/media-viewer.81fbeba4.js | 87.54kb | +0.00b | 10.69s | +4.20s ⚠️ |
| dist/card.c0473744.js | 76.79kb | +0.00b | 10.69s | +4.20s ⚠️ |
| dist/ui.ab1a2648.js | 73.70kb | +0.00b | 1.28m | +24.53s ⚠️ |
| dist/smartMediaEditor.e65bd2ee.js | 67.39kb | +0.00b | 1.28m | +24.53s ⚠️ |
| dist/component.10b390dc.js | 46.17kb | +0.00b | 58.99s | +25.64s ⚠️ |
| dist/DatePicker.0242ec52.js | 27.57kb | +0.00b | 1.07m | +25.45s ⚠️ |
| dist/EmojiPickerComponent.e7c7bb11.js | 26.74kb | +0.00b | 1.07m | +26.87s ⚠️ |
| dist/popup.4290aef9.js | 22.34kb | +0.00b | 57.91s | +25.49s ⚠️ |
| dist/popup.5b4d772e.js | 21.80kb | +0.00b | 57.91s | +25.49s ⚠️ |
| dist/popup.a791635d.js | 20.82kb | +0.00b | 57.91s | +25.49s ⚠️ |
| dist/js.16f29da2.js | 16.54kb | +0.00b | 1.98m | +20.47s ⚠️ |
| dist/popup.e0094d89.js | 12.77kb | +0.00b | 57.91s | +25.49s ⚠️ |
| dist/workerHasher.d7e20c98.js | 11.56kb | +0.00b | 1.10m | +26.58s ⚠️ |
| dist/EmojiTypeAheadComponent.1c09c30d.js | 10.44kb | +0.00b | 1.10m | +26.84s ⚠️ |
| dist/card.f99e1597.js | 5.89kb | +0.00b | 22.28s | +9.60s ⚠️ |
| dist/component.a26e593f.js | 5.62kb | +0.00b | 59.30s | +25.74s ⚠️ |
| dist/index.css | 3.38kb | +0.00b | 1.58m | +26.28s ⚠️ |
| dist/component.8fa44754.js | 3.38kb | +0.00b | 59.30s | +25.74s ⚠️ |
| dist/png-chunks-extract.3d8d8594.js | 3.33kb | +0.00b | 1.98m | +20.47s ⚠️ |
| dist/EmojiUploadComponent.0460cc59.js | 2.91kb | +0.00b | 1.10m | +26.84s ⚠️ |
| dist/component.ab65829f.js | 2.29kb | +0.00b | 59.30s | +25.74s ⚠️ |
| dist/ResourcedEmojiComponent.2d80adbd.js | 2.25kb | +0.00b | 10.63s | +4.20s ⚠️ |
| dist/media-picker-analytics-error-boundary.0f11d632.js | 1.85kb | +0.00b | 58.46s | +25.59s ⚠️ |
| dist/card.2c902906.js | 1.73kb | +0.00b | 22.28s | +9.59s ⚠️ |
| dist/16.dcacbe5c.js | 1.69kb | +0.00b | 59.84s | +25.87s ⚠️ |
| dist/date.cfe6153f.js | 1.63kb | +0.00b | 1.00m | +25.96s ⚠️ |
| dist/16.de3f3184.js | 1.62kb | +0.00b | 59.48s | +25.78s ⚠️ |
| dist/images.c60a97a9.js | 1.57kb | +0.00b | 1.00m | +25.95s ⚠️ |
| dist/feedback.01d7bcf7.js | 1.53kb | +0.00b | 1.07m | +26.87s ⚠️ |
| dist/workerHasher.854262ce.js | 1.48kb | +0.00b | 1.10m | +26.84s ⚠️ |
| dist/status.e3ed4764.js | 1.35kb | +0.00b | 1.01m | +26.24s ⚠️ |
| dist/list-number.dbe744df.js | 1.35kb | +0.00b | 1.01m | +26.06s ⚠️ |
| dist/16.f54dac28.js | 1.34kb | +0.00b | 59.67s | +25.82s ⚠️ |
| dist/16.8f0e8fe1.js | 1.29kb | +0.00b | 59.48s | +25.78s ⚠️ |
| dist/16.43b62c01.js | 1.29kb | +0.00b | 59.50s | +25.76s ⚠️ |
| dist/code.092e0137.js | 1.28kb | +0.00b | 1.00m | +25.91s ⚠️ |
| dist/16.b17a4d30.js | 1.27kb | +0.00b | 1.00m | +25.93s ⚠️ |
| dist/16.51bbc68e.js | 1.23kb | +0.00b | 59.85s | +25.88s ⚠️ |
| dist/component.8351f228.js | 1.23kb | +0.00b | 58.46s | +25.59s ⚠️ |
| dist/link.6fa27bb2.js | 1.20kb | +0.00b | 1.01m | +26.06s ⚠️ |
| dist/heading6.3996fe48.js | 1.19kb | +0.00b | 1.01m | +26.32s ⚠️ |
| dist/16.385d3ff5.js | 1.19kb | +0.00b | 59.84s | +25.87s ⚠️ |
| dist/heading3.9fea5413.js | 1.18kb | +0.00b | 1.01m | +26.33s ⚠️ |
| dist/16.b3ce4050.js | 1.16kb | +0.00b | 59.48s | +25.78s ⚠️ |
| dist/16.e37bbe6b.js | 1.15kb | +0.00b | 59.84s | +25.87s ⚠️ |
| dist/16.f7fe274a.js | 1.14kb | +0.00b | 1.00m | +25.93s ⚠️ |
| dist/emoji.c4bc5355.js | 1.12kb | +0.00b | 1.00m | +25.96s ⚠️ |
| dist/16.ac2ff83f.js | 1.10kb | +0.00b | 59.68s | +25.82s ⚠️ |
| dist/16.4c652b29.js | 1.10kb | +0.00b | 59.66s | +25.82s ⚠️ |
| dist/16.24165259.js | 1.09kb | +0.00b | 59.49s | +25.80s ⚠️ |
| dist/16.74510c56.js | 1.09kb | +0.00b | 59.66s | +25.82s ⚠️ |
| dist/16.97c5bdf9.js | 1.09kb | +0.00b | 59.68s | +25.83s ⚠️ |
| dist/16.a79db876.js | 1.09kb | +0.00b | 59.33s | +25.73s ⚠️ |
| dist/heading5.2f045997.js | 1.06kb | +0.00b | 1.01m | +26.33s ⚠️ |
| dist/16.6f8ac62a.js | 1.05kb | +0.00b | 59.31s | +25.74s ⚠️ |
| dist/expand.13380081.js | 1.05kb | +0.00b | 1.07m | +26.87s ⚠️ |
| dist/card.aae6ebb9.js | 1.04kb | +0.00b | 22.28s | +9.60s ⚠️ |
| dist/16.3e36da0b.js | 1.01kb | +0.00b | 59.85s | +25.89s ⚠️ |
| dist/heading2.2194210a.js | 1.00kb | +0.00b | 1.01m | +26.33s ⚠️ |
| dist/mention.9fdad24e.js | 981.00b | +0.00b | 1.01m | +26.05s ⚠️ |
| dist/fallback.305c8864.js | 980.00b | +0.00b | 1.01m | +26.24s ⚠️ |
| dist/heading4.63d1ca06.js | 976.00b | +0.00b | 1.01m | +26.33s ⚠️ |
| dist/layout.6f66c2da.js | 962.00b | +0.00b | 1.01m | +26.06s ⚠️ |
| dist/divider.160f3108.js | 940.00b | +0.00b | 1.00m | +25.96s ⚠️ |
| dist/quote.88bfdca4.js | 936.00b | +0.00b | 1.01m | +26.24s ⚠️ |
| dist/action.146450ba.js | 919.00b | +0.00b | 1.00m | +25.93s ⚠️ |
| dist/decision.06254b93.js | 898.00b | +0.00b | 1.00m | +25.96s ⚠️ |
| dist/panel-warning.59c8ef3f.js | 891.00b | +0.00b | 1.01m | +26.15s ⚠️ |
| dist/list.4f623de3.js | 870.00b | +0.00b | 1.01m | +26.06s ⚠️ |
| dist/heading1.65fb8482.js | 864.00b | +0.00b | 1.01m | +26.24s ⚠️ |
| dist/panel-error.c2479698.js | 791.00b | +0.00b | 1.01m | +26.15s ⚠️ |
| dist/panel.23cd7224.js | 788.00b | +0.00b | 1.01m | +26.14s ⚠️ |
| dist/table.4ace5843.js | 780.00b | +0.00b | 1.01m | +26.24s ⚠️ |
| dist/media-viewer-analytics-error-boundary.dd752a9e.js | 777.00b | +0.00b | 22.28s | +9.60s ⚠️ |
| dist/panel-success.6a7b97d7.js | 728.00b | +0.00b | 1.01m | +26.15s ⚠️ |
| dist/panel-note.5ff85bdc.js | 727.00b | +0.00b | 1.01m | +26.14s ⚠️ |
| dist/simpleHasher.884d4eb6.js | 446.00b | +0.00b | 1.10m | +26.59s ⚠️ |
| dist/esm.976debcc.js | 178.00b | +0.00b | 10.50s | +4.13s ⚠️ |
| dist/dropzone.976debcc.js | 177.00b | +1.00b ⚠️ | 1.98m | +1.44m ⚠️ |
| dist/clipboard.976debcc.js | 177.00b | +1.00b ⚠️ | 58.42s | +25.55s ⚠️ |
| dist/browser.976debcc.js | 177.00b | +1.00b ⚠️ | 1.28m | +44.13s ⚠️ |
| dist/editorView.976debcc.js | 177.00b | -1.00b 🚀 | 57.92s | +4.37s ⚠️ |
| dist/media-card-analytics-error-boundary.976debcc.js | 176.00b | -5.00b 🚀 | 1.00m | -38.18s 🚀 |
| dist/16.976debcc.js | 174.00b | -5.00b 🚀 | 58.42s | +24.34s ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 2.33mb | +6.00b ⚠️ | 550.00ms | +207.00ms ⚠️ |
| dist/pdfRenderer.85e56a51.js | 1.11mb | +0.00b | 365.00ms | +141.00ms ⚠️ |
| dist/smartMediaEditor.678dca17.js | 601.49kb | +0.00b | 221.00ms | +60.00ms ⚠️ |
| dist/popup.cc1a1fa6.js | 285.42kb | +0.00b | 232.00ms | +60.00ms ⚠️ |
| dist/EmojiPickerComponent.7202df5c.js | 105.88kb | +0.00b | 159.00ms | +25.00ms ⚠️ |
| dist/Toolbar.5c7a6fbb.js | 105.62kb | +0.00b | 155.00ms | +30.00ms ⚠️ |
| dist/media-viewer.81fbeba4.js | 87.54kb | +0.00b | 205.00ms | +44.00ms ⚠️ |
| dist/card.c0473744.js | 76.79kb | +0.00b | 206.00ms | +46.00ms ⚠️ |
| dist/ui.ab1a2648.js | 73.71kb | +0.00b | 156.00ms | +24.00ms ⚠️ |
| dist/smartMediaEditor.e65bd2ee.js | 67.39kb | +0.00b | 156.00ms | +24.00ms ⚠️ |
| dist/component.10b390dc.js | 46.17kb | +0.00b | 179.00ms | +33.00ms ⚠️ |
| dist/DatePicker.0242ec52.js | 27.57kb | +0.00b | 142.00ms | +13.00ms ⚠️ |
| dist/EmojiPickerComponent.e7c7bb11.js | 26.74kb | +0.00b | 143.00ms | +14.00ms ⚠️ |
| dist/popup.4290aef9.js | 22.34kb | +0.00b | 181.00ms | +32.00ms ⚠️ |
| dist/popup.5b4d772e.js | 21.80kb | +0.00b | 180.00ms | +32.00ms ⚠️ |
| dist/popup.a791635d.js | 20.82kb | +0.00b | 183.00ms | +35.00ms ⚠️ |
| dist/js.16f29da2.js | 16.54kb | +0.00b | 181.00ms | +31.00ms ⚠️ |
| dist/popup.e0094d89.js | 12.77kb | +0.00b | 181.00ms | +33.00ms ⚠️ |
| dist/workerHasher.d7e20c98.js | 11.56kb | +0.00b | 141.00ms | +13.00ms ⚠️ |
| dist/EmojiTypeAheadComponent.1c09c30d.js | 10.44kb | +0.00b | 142.00ms | +14.00ms ⚠️ |
| dist/card.f99e1597.js | 5.89kb | +0.00b | 182.00ms | +31.00ms ⚠️ |
| dist/component.a26e593f.js | 5.62kb | +0.00b | 178.00ms | +34.00ms ⚠️ |
| dist/index.css | 3.38kb | +0.00b | 114.00ms | +10.00ms ⚠️ |
| dist/component.8fa44754.js | 3.38kb | +0.00b | 179.00ms | +34.00ms ⚠️ |
| dist/png-chunks-extract.3d8d8594.js | 3.33kb | +0.00b | 181.00ms | +31.00ms ⚠️ |
| dist/EmojiUploadComponent.0460cc59.js | 2.91kb | +0.00b | 143.00ms | +15.00ms ⚠️ |
| dist/component.ab65829f.js | 2.29kb | +0.00b | 178.00ms | +33.00ms ⚠️ |
| dist/ResourcedEmojiComponent.2d80adbd.js | 2.25kb | +0.00b | 185.00ms | +32.00ms ⚠️ |
| dist/media-picker-analytics-error-boundary.0f11d632.js | 1.85kb | +0.00b | 179.00ms | +29.00ms ⚠️ |
| dist/card.2c902906.js | 1.73kb | +0.00b | 182.00ms | +31.00ms ⚠️ |
| dist/16.dcacbe5c.js | 1.69kb | +0.00b | 168.00ms | +30.00ms ⚠️ |
| dist/date.cfe6153f.js | 1.63kb | +0.00b | 178.00ms | +43.00ms ⚠️ |
| dist/16.de3f3184.js | 1.62kb | +0.00b | 176.00ms | +33.00ms ⚠️ |
| dist/images.c60a97a9.js | 1.57kb | +0.00b | 160.00ms | +25.00ms ⚠️ |
| dist/feedback.01d7bcf7.js | 1.53kb | +0.00b | 150.00ms | +20.00ms ⚠️ |
| dist/workerHasher.854262ce.js | 1.48kb | +0.00b | 141.00ms | +13.00ms ⚠️ |
| dist/status.e3ed4764.js | 1.35kb | +0.00b | 152.00ms | +21.00ms ⚠️ |
| dist/list-number.dbe744df.js | 1.35kb | +0.00b | 155.00ms | +21.00ms ⚠️ |
| dist/16.f54dac28.js | 1.34kb | +0.00b | 169.00ms | +29.00ms ⚠️ |
| dist/16.8f0e8fe1.js | 1.29kb | +0.00b | 177.00ms | +34.00ms ⚠️ |
| dist/16.43b62c01.js | 1.29kb | +0.00b | 172.00ms | +31.00ms ⚠️ |
| dist/code.092e0137.js | 1.28kb | +0.00b | 161.00ms | +25.00ms ⚠️ |
| dist/16.b17a4d30.js | 1.27kb | +0.00b | 161.00ms | +24.00ms ⚠️ |
| dist/16.51bbc68e.js | 1.23kb | +0.00b | 161.00ms | +24.00ms ⚠️ |
| dist/component.8351f228.js | 1.23kb | +0.00b | 178.00ms | +32.00ms ⚠️ |
| dist/link.6fa27bb2.js | 1.20kb | +0.00b | 159.00ms | +24.00ms ⚠️ |
| dist/heading6.3996fe48.js | 1.19kb | +0.00b | 150.00ms | +20.00ms ⚠️ |
| dist/16.385d3ff5.js | 1.19kb | +0.00b | 162.00ms | +24.00ms ⚠️ |
| dist/heading3.9fea5413.js | 1.18kb | +0.00b | 151.00ms | +20.00ms ⚠️ |
| dist/16.b3ce4050.js | 1.16kb | +0.00b | 174.00ms | +32.00ms ⚠️ |
| dist/16.e37bbe6b.js | 1.15kb | +0.00b | 162.00ms | +24.00ms ⚠️ |
| dist/16.f7fe274a.js | 1.14kb | +0.00b | 161.00ms | +25.00ms ⚠️ |
| dist/emoji.c4bc5355.js | 1.12kb | +0.00b | 159.00ms | +18.00ms ⚠️ |
| dist/16.ac2ff83f.js | 1.10kb | +0.00b | 168.00ms | +29.00ms ⚠️ |
| dist/16.4c652b29.js | 1.10kb | +0.00b | 172.00ms | +31.00ms ⚠️ |
| dist/16.24165259.js | 1.09kb | +0.00b | 172.00ms | +30.00ms ⚠️ |
| dist/16.74510c56.js | 1.09kb | +0.00b | 169.00ms | +29.00ms ⚠️ |
| dist/16.97c5bdf9.js | 1.09kb | +0.00b | 168.00ms | +29.00ms ⚠️ |
| dist/16.a79db876.js | 1.09kb | +0.00b | 177.00ms | +34.00ms ⚠️ |
| dist/heading5.2f045997.js | 1.06kb | +0.00b | 150.00ms | +20.00ms ⚠️ |
| dist/16.6f8ac62a.js | 1.05kb | +0.00b | 177.00ms | +33.00ms ⚠️ |
| dist/expand.13380081.js | 1.05kb | +0.00b | 144.00ms | +15.00ms ⚠️ |
| dist/card.aae6ebb9.js | 1.04kb | +0.00b | 182.00ms | +31.00ms ⚠️ |
| dist/16.3e36da0b.js | 1.01kb | +0.00b | 162.00ms | +25.00ms ⚠️ |
| dist/heading2.2194210a.js | 1.00kb | +0.00b | 151.00ms | +21.00ms ⚠️ |
| dist/mention.9fdad24e.js | 981.00b | +0.00b | 154.00ms | +20.00ms ⚠️ |
| dist/fallback.305c8864.js | 980.00b | +0.00b | 152.00ms | +21.00ms ⚠️ |
| dist/heading4.63d1ca06.js | 976.00b | +0.00b | 150.00ms | +19.00ms ⚠️ |
| dist/layout.6f66c2da.js | 962.00b | +0.00b | 160.00ms | +25.00ms ⚠️ |
| dist/divider.160f3108.js | 940.00b | +0.00b | 160.00ms | +18.00ms ⚠️ |
| dist/quote.88bfdca4.js | 936.00b | +0.00b | 152.00ms | +21.00ms ⚠️ |
| dist/action.146450ba.js | 919.00b | +0.00b | 179.00ms | +43.00ms ⚠️ |
| dist/decision.06254b93.js | 898.00b | +0.00b | 160.00ms | +24.00ms ⚠️ |
| dist/panel-warning.59c8ef3f.js | 891.00b | +0.00b | 153.00ms | +21.00ms ⚠️ |
| dist/list.4f623de3.js | 870.00b | +0.00b | 155.00ms | +21.00ms ⚠️ |
| dist/heading1.65fb8482.js | 864.00b | +0.00b | 152.00ms | +21.00ms ⚠️ |
| dist/panel-error.c2479698.js | 791.00b | +0.00b | 154.00ms | +21.00ms ⚠️ |
| dist/panel.23cd7224.js | 788.00b | +0.00b | 152.00ms | +20.00ms ⚠️ |
| dist/table.4ace5843.js | 780.00b | +0.00b | 152.00ms | +21.00ms ⚠️ |
| dist/media-viewer-analytics-error-boundary.dd752a9e.js | 777.00b | +0.00b | 183.00ms | +32.00ms ⚠️ |
| dist/panel-success.6a7b97d7.js | 728.00b | +0.00b | 153.00ms | +21.00ms ⚠️ |
| dist/panel-note.5ff85bdc.js | 727.00b | +0.00b | 154.00ms | +21.00ms ⚠️ |
| dist/simpleHasher.884d4eb6.js | 446.00b | +0.00b | 141.00ms | +14.00ms ⚠️ |
| dist/esm.976debcc.js | 176.00b | +2.00b ⚠️ | 184.00ms | +31.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.976debcc.js | 176.00b | +2.00b ⚠️ | 181.00ms | +31.00ms ⚠️ |
| dist/dropzone.976debcc.js | 176.00b | +2.00b ⚠️ | 183.00ms | +35.00ms ⚠️ |
| dist/clipboard.976debcc.js | 176.00b | +2.00b ⚠️ | 179.00ms | +32.00ms ⚠️ |
| dist/browser.976debcc.js | 176.00b | +2.00b ⚠️ | 179.00ms | +32.00ms ⚠️ |
| dist/16.976debcc.js | 176.00b | +2.00b ⚠️ | 161.00ms | +25.00ms ⚠️ |
| dist/editorView.976debcc.js | 176.00b | +2.00b ⚠️ | 139.00ms | +19.00ms ⚠️ |

</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/254663202911027721)**

**Comment by DeMoorJasper** [comment]
From the benchmarks it appears this has a quite significant overhead, maybe this isn't such a good idea after all.

**Comment by DeMoorJasper** [comment]
Gonna close this due to it not really being a good idea after all...

**Pull request closed without merging**